content/en/news/releases/1.6.x/announcing-1.6.x/upgrade-notes/index.md

```diff
@@ -0,0 +1,126 @@
+$ kubectl get policies.authentication.istio.io --all-namespaces
+$ kubectl get meshpolicies.authentication.istio.io --all-namespaces
+$ kubectl get rbacconfigs.rbac.istio.io --all-namespaces
+$ kubectl get clusterrbacconfigs.rbac.istio.io --all-namespaces
+$ kubectl get serviceroles.rbac.istio.io --all-namespaces
+$ kubectl get servicerolebindings.rbac.istio.io --all-namespaces
+{{< /text >}}
+
+If there are any `v1alpha1` security policies in your clusters, migrate to the new APIs before upgrading.
+
+To ensure that `v1alpha1` security policies aren't applied in the future, delete the Custom Resource Definitions (CRDs) using the `v1alpha1` security policy APIs with the following commands:
+
+{{< text bash >}}
+$ kubectl delete crd policies.authentication.istio.io
+$ kubectl delete crd meshpolicies.authentication.istio.io
+$ kubectl delete crd rbacconfigs.rbac.istio.io
+$ kubectl delete crd clusterrbacconfigs.rbac.istio.io
+$ kubectl delete crd serviceroles.rbac.istio.io
+$ kubectl delete crd servicerolebindings.rbac.istio.io
+{{< /text >}}
+
+## Istio configuration during installation
+
+Past Istio releases deployed configuration objects during installation. The presence of those objects caused the following issues:
+
+- Problems with upgrades
+- A confusing user experience
+- A less flexible installation
+
+To address these issues, Istio 1.6 minimized the configuration objects deployed during installation.
+
+The following configurations are impacted:
+
+- `global.mtls.enabled`: Configuration removed to avoid confusion. Configure a peer authentication policy to enable [strict mTLS](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode) instead.
+- No default `Gateway` and associated `Certificate` custom resources are deployed during installation. Go to the [Ingress task](/docs/tasks/traffic-management/ingress/) to configure a gateway for your mesh.
+- Istio no longer creates `Ingress` custom resources  for telemetry addons. Visit [remotely accessing telemetry addons](/docs/tasks/observability/gateways/) to learn how to reach the addons externally.
+- The default sidecar configuration is no longer defined through the automatically generated `Sidecar` custom resource. The default configuration is implemented internally and the change should have no impact on deployments.
+
+## Reach Istiod through external workloads
+
+Istio 1.6 configures Istiod to be `cluster-local` by default. This means that an Istio gateway will only send traffic to an Istiod instance located in the same cluster. Workloads external to the cluster must now access Istiod via the ingress gateway. This configuration change was required to support multicluster deployments with master and remote clusters. Setting Istiod to `cluster-local` prevents the master cluster ingress gateway from seeing the Istiod endpoints in the remote cluster. The Istio team is actively investigating alternatives which do not require a cluster-local Istiod.
```

Comment:
```suggestion
In Istio 1.6, the `cluster-local` configuration of Istiod is enabled by default.  With `cluster-local` enabled, an Istio gateway only sends traffic to an Istiod instance on the same cluster. Workloads on another cluster can only access the  Istiod instance through the Istio gateway. This configuration change supports multicluster deployments with master and remote clusters. This configuration prevents the ingress gateway of the master cluster from seeing the Istiod endpoints of the remote cluster. The Istio team is actively investigating alternatives to no longer require `cluster-local`.
```

Reply:
@rcaballeromx 

> With `cluster-local` enabled, an Istio gateway only sends traffic to an Istiod instance on the same cluster

This isn't quite accurate... it's more than that. This should be `With cluster-local enabled, Istiod will only be reachable from workloads residing within the same cluster`.

Also, we should probably expand on this sentence a bit: `This configuration prevents the ingress gateway of the master cluster from incorrectly forwarding service discovery requests to the Istiod in the remote cluster. The Istio team is actively investigating alternatives which will no longer require a cluster-local Istiod for master/remote deployments.`

Reply:
@nmittler 
For the first, that's fine as long as it is written more clearly:
`With cluster-local enabled, only workloads running on the same cluster can reach Istiod.`
I'm also fine with expanding, as long as the language is cleaned a bit too:
`This configuration prevents the ingress gateway of the master cluster from incorrectly forwarding service discovery requests to the Istiod in the remote cluster. The Istio team is actively investigating alternatives which no longer require a cluster-local Istiod for master/remote deployments.`

Reply:
suggestions lgtm

Reply:
Adding. Thanks. To clarify, does this mean that Istiod will only accept traffic from within the same cluster? And this is *required* for multicluster to work, correct? I think that's what I understood before, just want to make sure.

Reply:
Also, we say that it supports multicluster deployments with master and remote clusters. If I understand correctly, it doesn't support multicluster deployments without master and remote clusters or single cluster deployments. Is that correct?

Reply:
@nmittler

Reply:
@brian-avery 

> Adding. Thanks. To clarify, does this mean that Istiod will only accept traffic from within the same cluster? And this is required for multicluster to work, correct? I think that's what I understood before, just want to make sure.

Yes, that's right. It's explained in the line `This configuration prevents the ingress gateway of the master cluster from incorrectly forwarding service discovery requests to the Istiod in the remote cluster.`

> Also, we say that it supports multicluster deployments with master and remote clusters. If I understand correctly, it also supports multicluster deployments without master and remote clusters or single cluster deployments. Is that correct? Should we clarify?

Things are sort of broken with multi-master deployments as well, although not so bad. If the ingress round-robins discovery requests between the masters it's generally ok, unless they are way out of sync with each other. The bigger issue is for remote clusters, since they have no discovery information available and if an Envoy connects to it, badness ensues.

Reply:
Thanks.  That makes sense. Also clarifying that the configuration change is required for multicluster